PhotoCollectionView/Sourses/Controllers/AlbumScreen/Controller/AlbumsViewController.swift

```diff
@@ -0,0 +1,158 @@
+    }()
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        title = "Albums"
+        navigationController?.navigationBar.prefersLargeTitles = true
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: nil)
+        setupHierarchy()
+        setupCollection()
+    }
+    
+    // MARK: - Setup Layout
+    
+    private func setupHierarchy() {
+        view.addSubview(collectionView)
+    }
+    
+    private func setupCollection() {
+        collectionView.snp.makeConstraints { make in
+            make.top.bottom.left.right.equalTo(view)
+        }
+    }
+}
+
+extension AlbumsViewController {
+    private func createLayout() -> UICollectionViewCompositionalLayout {
+        return UICollectionViewCompositionalLayout { (section, layoutEnvironment: NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection in
+            
+            func setupAlbumsSections(count: Int, width: CGFloat) -> NSCollectionLayoutSection{
+                let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1),heightDimension: .fractionalHeight(1))
+                
+                let layoutItem = NSCollectionLayoutItem(layoutSize: itemSize)
+                layoutItem.contentInsets = NSDirectionalEdgeInsets(top: 5, leading: 10, bottom: 5, trailing: 0)
+                
+                let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1 / 2.1), heightDimension: .fractionalWidth(width))
+                let layoutGroup = NSCollectionLayoutGroup.vertical(layoutSize: groupSize, subitem: layoutItem, count: count)
+                
+                layoutGroup.contentInsets = NSDirectionalEdgeInsets(top: 0, leading: 5, bottom: 0, trailing: 0)
+                
+                let layoutSection = NSCollectionLayoutSection(group: layoutGroup)
+                layoutSection.contentInsets = NSDirectionalEdgeInsets(top: 0, leading: 5, bottom: 0, trailing: 5)
+                layoutSection.orthogonalScrollingBehavior = .groupPaging
+                
+                let layoutSectionHeaderSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(0.93), heightDimension: .estimated(50))
+                let layoutSectionHeader = NSCollectionLayoutBoundarySupplementaryItem(layoutSize: layoutSectionHeaderSize, elementKind: UICollectionView.elementKindSectionHeader, alignment: .top)
+                layoutSection.boundarySupplementaryItems = [layoutSectionHeader]
+                layoutSection.contentInsets = NSDirectionalEdgeInsets(top: 0, leading: 0, bottom: 20, trailing: 0)
+                
+                return layoutSection
+            }
+            
+            switch section {
+            case 0:
+                return setupAlbumsSections(count: 2, width: 1 / 1.8 * 2)
+            case 1:
+                return setupAlbumsSections(count: 1, width: 1 / 1.8)
```

Comment:
То что нужно, но вот этот момент 1 / 1.8 всегда одинаковый, а вот множетель разный в первом случае 2 в другом просто 1
```suggestion
            case 0:
                return setupAlbumsSections(count: 2, width: 1 / 1.8 * 2)
            case 1:
                return setupAlbumsSections(count: 1, width: 1 / 1.8)
```